opencanary/logger.py

```diff
@@ -99,7 +99,8 @@ class LoggerBase(object):
 
     def sanitizeLog(self, logdata):
         logdata['node_id'] = self.node_id
-        logdata['local_time'] = datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S.%f")
+        logdata['UTC_time'] = datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S.%f")
+        logdata['local_time'] = datetime.now().strftime("%Y-%m-%d %H:%M:%S.%f")
```

Comment:
maybe we go for something like `logdata['local_time_adjusted']`

Reply:
OK I see. This way it will be obvious that something has changed.

Reply:
See the updated files.